### PR TITLE
118 option wegboeken

### PR DESCRIPTION
**Pull request by SimplyPancake**
**Please describe what your PR is fixing**
Adds a write off option, where it can be enabled in the django admin per organisation.
When it is enabled, writeoff categories can be added in the admin panel of the organisation.
Then, bartenders are able to select 'write off' in the juliana POS, where they can put their orders on a category.
Then, in the sales overview, a tab has been added for the writeoff categories, where it shows how much (and which items) have been written off per category. A JSON export is also available.

Concretely, which issues does your PR solve?
#118 

**Does your PR include any django migrations?**
yes!

PR does not include CSS changes

### CODE REVIEW

File: assets/js/juliana.js [hunk 1]
```diff
@@ -302,6 +319,27 @@ Receipt = {
...
+    writeoffNow: function (categoryId) {
+        let rpcRequest = {
+            jsonrpc: '2.0',
+            method: 'juliana.writeoff.save',
+            params: {
+                event_id: Settings.event_id,
+                writeoff_id: categoryId,
+                purchases: Receipt.receipt,
+            },
+            id: 2 // id used for?
```
Comment (SimplyPancake) [inline assets/js/juliana.js:332]: Still not sure what this ID is used for.

Comment (Kurocon) [reply assets/js/juliana.js:332]: This `id` field is part of the JSONRPC spec, and must be a client-specified identifier for the request.
The current value `2` is probably fine for our use-case.
See also the IA API docs: https://www.inter-actief.utwente.nl/api/docs/#header_jsonrpcrequest

**Review by Kurocon — APPROVED** [review]
Overall the feature looks really good, good work!

I've added one commit with some of my own improvements, see the commit message as to what I've changed. If you agree with the changes, then I am OK with merging this for further testing on staging.

**Comment by SimplyPancake** [comment]
I checked it out and I agree with the changes! Thanks for the 'small' changes ;) The changes make it more coherent, looks good!

**Pull request merged**